config.m4

```diff
@@ -282,9 +322,9 @@ if test "$PHP_MONGODB" != "no"; then
     AC_SUBST(MONGOC_ENABLE_RDTSCP, 0)
     AC_SUBST(MONGOC_ENABLE_SHM_COUNTERS, 0)
     AC_SUBST(MONGOC_TRACE, 1)
-
-    dnl TODO: Replace with detection for libmongocrypt for PHPC-1293
-    AC_SUBST(MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION, 0)
+    AC_SUBST(MONGOCRYPT_ENABLE_TRACE, 1)
+    AC_SUBST(MONGOCRYPT_IS_POSIX, 1)
```

Comment:
I'll note that you're always defining `IS_POSIX` as `1` and `IS_WIN` as `0`, so that may support my later suggestion to not check `$TARGET_OS` and always compile `PHP_MONGODB_MONGOCRYPT_POSIX_SOURCES`.